migration.md

```diff
@@ -417,6 +417,38 @@ The standard library provides multiple options for threads, locks, and mutexes a
 | `sf::ThreadLocal`    | `thread_local`                                   |
 | `sf::ThreadLocalPtr` | `thread_local`                                   |
 
+## Sound Samples and Channel Map
+
+SFML 3 introduces the concept of a Channel Map which defines the mapping of the position in sample frame to the sound channel.
+For example, if you have a sound frame with six different samples for a 5.1 sound system, the Channel Map defines how each of those samples map to which speaker channel.
+
+SFML 2 always assumed the order as specified by OpenAL.
+
+```cpp
+auto samples = std::vector<std::int16_t>();
+// ...
+
+auto channelMap = std::vector<sf::SoundChannel>{
+    sf::SoundChannel::FrontLeft,
+    sf::SoundChannel::FrontCenter,
+    sf::SoundChannel::FrontRight,
+    sf::SoundChannel::BackRight,
+    sf::SoundChannel::BackCenter,
+    sf::SoundChannel::BackLeft,
+    sf::SoundChannel::LowFrequencyEffects
+};
+auto soundBuffer = sf::SoundBuffer(samples.data(), samples.size(), 6, 44100, channelMap);
```

Comment:
The number of entries in `channelMap` should match the `channelCount` argument. 6 != 7

Reply:
Or rather, since I wanted a 5.1 system, I shouldn't add a back center speaker 🫠

Reply:
See #3351